app/controllers/home-office-details.ts

```diff
@@ -47,23 +53,93 @@ function postDateLetterSent(req: Request, res: Response, next: NextFunction) {
         errorList: Object.values(validation)
       });
     }
+    const { day, month, year } = req.body;
+    const diffInDays = moment().diff(moment(`${year} ${month} ${day}`, 'YYYY MM DD'), 'days');
+    if (diffInDays < 0) {
+      const error = {
+        text: 'The date is in the future, please re-enter the date.',
```

Comment:
Should this error come from the `i18n.validationErrors` in the `en.json` instead of hard-coding it?